src/main/java/com/minecolonies/coremod/entity/ai/citizen/baker/BakerRecipes.java

```diff
@@ -0,0 +1,91 @@
+package com.minecolonies.coremod.entity.ai.citizen.baker;
+
+import com.minecolonies.coremod.entity.ai.util.RecipeStorage;
+import net.minecraft.init.Blocks;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.init.Blocks'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)